packages/patterns/src/generatePatternStyles.ts

```diff
@@ -46,23 +59,26 @@ function truncate(val, pos) {
  * @param {Object} pattern.size Gradient size if not full size
  * @return {string} Minimal description for gradient.
  */
-function getGradientDescription({ type, rotation, center, size }) {
-  const sizeString = size
-    ? `ellipse ${truncate(100 * size.w, 2)}% ${truncate(100 * size.h, 2)}%`
-    : '';
-  const centerString = center
-    ? ` at ${truncate(100 * center.x, 2)}% ${truncate(100 * center.y, 2)}%`
-    : '';
-  switch (type) {
-    case 'radial':
+function getGradientDescription(pattern: Gradient) {
+  switch (pattern.type) {
+    case 'radial': {
```

Comment:
Should we use the `PatternType` enum here for the cases?

Reply:
Oh yes, my bad.